src/components/error-message/index.md.njk

```diff
@@ -144,6 +140,20 @@ Find out how often people see them. This will let you:
 - A/B test variations
 - redesign a journey
 
+## Use localisation to translate ‘error message’ into other languages
+
+Read how to [set up localisation with Nunjucks and Javascript](/get-started/localisation/).
```

Comment:
Change sentence to remove 'set up'. Maybe to 'Read how to use localisation with...'